### PR TITLE
Add hint for Tag list input

### PR DESCRIPTION
**Pull request by damianpm**
**What this PR does / why we need it**:

- Adding a hint for the **Tag list** input, in Developer portal **Pages** and **Files**
- Add tests for both pages
 
Closes https://github.com/3scale/porta/pull/2790

Screenshot 2021-11-29 at 12 10 46

Jira issues:

https://issues.redhat.com/browse/THREESCALE-7898
https://issues.redhat.com/browse/THREESCALE-7935

**Steps to reproduce:**

- Go to **Audience > Developer Portal > Content**
- Select a file (eg: mouse.jpg)
- Write some tags
- Click save

### CODE REVIEW

**Comment by akostadinov** [comment]
That's cool. Could you add some tests that tags are properly saved and later shown comma separated? e.g. in `features/old/cms/pages.feature` , for files I don't see an existing file, maybe I just can't spot.

**Review by hallelujah — APPROVED** [review]

File: features/old/cms/pages.feature [hunk 1]
```diff
@@ -39,7 +39,7 @@ Feature: CMS Pages
         | Path           | /potato       |
         | Content type   | text/css      |
         | Handler        | markdown      |
-        | Tag list       | potato salad |
+        | Tag list       | potato, salad |
```
Comment (hallelujah) [inline features/old/cms/pages.feature:42]: So before it was space separated list?
If so then this is a major change, we should verify that we do not break customer workflow

**Review by hallelujah — CHANGES_REQUESTED** [review]

File: features/old/cms/pages.feature (hunk 1 above)
Comment (akostadinov) [reply features/old/cms/pages.feature:42]: Before it showed space separated, but when you saved, it was saves as a single tag. So customer saves as "tag1, tag2, tag3", then on page reload it appeared as "tag1 tag2 tag3" and if saved again would save as a single tag "tag1 tag2 tag3". Basically now we avoid customer messing up tags by mistake and document how to split tags properly.

Comment (akostadinov) [reply features/old/cms/pages.feature:42]: see https://issues.redhat.com/browse/THREESCALE-7898

**Comment by akostadinov** [comment]
Looks great! Can you try to write some File UI test? I now found features/provider/cms/files.feature, maybe inside it.

**Review by akostadinov — DISMISSED** [review]
Looks nice. Thank you! could you point pull request to master branch and close mine?

**Comment by damianpm** [comment]
@akostadinov @hallelujah PR updated, please take a look

**Review by akostadinov — APPROVED** [review]
Thank you!

**Pull request merged**